app/models/challenge.rb

```diff
@@ -251,10 +251,10 @@ def can_create? cuser
   end
 
   def can_update? cuser
-    cuser and (contester2.team.is_leader? cuser or cuser.admin?) and status == STATUS_PENDING and autodefault.future?
+    cuser and (contester2.team.is_leader? cuser or cuser.admin?) and status == STATUS_PENDING# and autodefault.future?
```

Comment:
Line is too long. [118/80]<br>Use `&&` instead of `and`.<br>Use `||` instead of `or`.<br>Put a space before an end-of-line comment.